shap/explainers/_kernel.py

```diff
@@ -89,9 +89,9 @@ def __init__(self, model, data, feature_names=None, link=IdentityLink(), **kwarg
 
         # convert incoming inputs to standardized iml objects
         self.link = convert_to_link(link)
-        self.model = convert_to_model(model)
         self.keep_index = kwargs.get("keep_index", False)
         self.keep_index_ordered = kwargs.get("keep_index_ordered", False)
+        self.model = convert_to_model(model, self.keep_index)
```

Comment:
Nitpick: suggest passing this argument by keyword rather than by position, for slightly better clarity & robustness to future changes

```suggestion
        self.model = convert_to_model(model, keep_index=self.keep_index)
```